### PR TITLE
Add documentation about how to use Logfire with `StripeClient`

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: docs/integrations/stripe.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Stripe
+
+[Stripe] is a popular payment gateway that allows businesses to accept payments online.
```
Comment (alexmojaki) [inline docs/integrations/stripe.md:3]: should there be a link here?

File: docs/integrations/stripe.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+The stripe Python client has both synchronous and asynchronous methods for making requests to the Stripe API.
+
+By default, the stripe client uses the `requests` package for making synchronous requests and
```
Comment (alexmojaki) [inline docs/integrations/stripe.md:7]: wtf stripe...

File: docs/integrations/stripe.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+async def main():
+    await client.customers.list_async()  # uses `httpx`
+
+if __name__ == '__main__':
+    import asyncio
+
+    asyncio.run(main())
+```
+
+You read more about this on the [Configuring an HTTP Client] section on the stripe repository.
```
Comment (alexmojaki) [inline docs/integrations/stripe.md:28]: link?

File: docs/integrations/stripe.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+By default, the stripe client uses the `requests` package for making synchronous requests and
+the `httpx` package for making asynchronous requests.
+
+```py
+from stripe import StripeClient
+
+client = StripeClient(api_key='<your_secret_key>')
+
+# Synchronous request
+client.customers.list()  # uses `requests`
```
Comment (alexmojaki) [inline docs/integrations/stripe.md:16]: is this code block helpful? i think maybe just skip to the instrumenting part

File: docs/integrations/stripe.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    asyncio.run(main())
+```
+
+You read more about this on the [Configuring an HTTP Client] section on the stripe repository.
+
+## Synchronous Requests
+
+As mentioned, by default, `stripe` uses the `requests` package for making HTTP requests.
+
+In this case, you'll need to call [`logfire.instrument_requests()`][logfire.Logfire.instrument_requests].
```
Comment (alexmojaki) [inline docs/integrations/stripe.md]: rather link to the guide, same for httpx

File: docs/integrations/stripe.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+```py
+import os
+from logging import basicConfig
```
Comment (alexmojaki) [inline docs/integrations/stripe.md]: unused

File: docs/integrations/stripe.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+import logfire
+from stripe import StripeClient
+
+logfire.configure()
+logfire.instrument_requests()
+
+client = StripeClient(api_key=os.getenv('STRIPE_SECRET_KEY'))
+
+with logfire.span('list customers'):
```
Comment (alexmojaki) [inline docs/integrations/stripe.md]: is this helpful?

File: docs/integrations/stripe.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+    asyncio.run(main())
+```
+
+!!! note
+    If you use the `http_client` parameter to configure the stripe client to use a different HTTP client,
+    you'll need to call the appropriate instrumentation method.
+
+## Add logging instrumentation
+
+Stripe also has a logger (`logger = getLogger('stripe')`) that you can instrument with **Logfire**.
```
Comment (alexmojaki) [inline docs/integrations/stripe.md]: link to the logging guide

**Review by alexmojaki — CHANGES_REQUESTED** [review]

File: docs/integrations/stripe.md (hunk 1 above)
Comment (Kludex) [reply docs/integrations/stripe.md:3]: It does have. You didn't run this locally.

File: docs/integrations/stripe.md (hunk 3 above)
Comment (Kludex) [reply docs/integrations/stripe.md:28]: It does have already.

File: docs/integrations/stripe.md (hunk 1 above)
Comment (alexmojaki) [reply docs/integrations/stripe.md:3]: is the cloudflare preview not reliable?

Comment (Kludex) [reply docs/integrations/stripe.md:3]: Ok... This is my bad!

I must have deleted the [Stripe] link at some point during development. Cloudflare is good, and your comment is good.

File: docs/integrations/stripe.md (hunk 2 above)
Comment (Kludex) [reply docs/integrations/stripe.md:7]: Yep.

File: docs/integrations/stripe.md (hunk 4 above)
Comment (Kludex) [reply docs/integrations/stripe.md:16]: Yes. I like to have snippets where I can copy paste and see what happens...

File: docs/integrations/stripe.md (hunk 3 above)
Comment (alexmojaki) [reply docs/integrations/stripe.md:28]: still missing

Comment (Kludex) [reply docs/integrations/stripe.md:28]: Sorry.

**Review by alexmojaki — APPROVED** [review]

**Pull request merged**